src/core/diagram/qgspiediagram.h

```diff
@@ -38,6 +36,8 @@ class QgsRenderContext;
 class CORE_EXPORT QgsPieDiagram: public QgsDiagram SIP_NODEFAULTCTORS
 {
   public:
+    static const QString DIAGRAM_NAME_PIE;
```

Comment:
```suggestion
    static const QString DIAGRAM_NAME_PIE SIP_SKIP;
```
can you do the same for all the others?